app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] One of the requirements for this milestone is to design the `login` page to look identical to that provided in the wireframe. Currently, the input boxes and the submit button is not identical to the view in the wireframe. Kindly redesign it properly. 👍 💻 

### Your Design

![image](https://github.com/tomdan-ai/Gem-Blog/assets/77106962/47f565f1-3e84-4770-b523-2800557758d6)

### Wireframe Template

![image](https://github.com/tomdan-ai/Gem-Blog/assets/77106962/d2be5d27-69d2-449d-97a9-f8f01d516cba)